pwa/components/admin/authProvider.tsx

```diff
@@ -42,7 +42,23 @@ const authProvider: AuthProvider = {
 
     return Promise.resolve();
   },
-  getPermissions: () => Promise.resolve(),
+  getPermissions: async () => {
```

Comment:
In general, the user permissions are persisted during login, to avoid calling the auth provider backend on every page. That way, `getPermissions` can return the cached permissions (e.g. stored in a variable local to the authProvider).